spec/helpers/holding_block_spec.rb

```diff
@@ -240,6 +240,24 @@
       end
     end
 
+    context '#holding_block_search with links only' do
+      let(:document) do
```

Comment:
Can you update the test title :...links from the electronic_access_1display.. 
because we have links from other fields as well.

Reply:
Ignore the title is ok. We can add more tests later regarding other fields that are also used in the online display.